yal/quiz.py

```diff
@@ -138,18 +138,27 @@ async def state_quiz_answer(self):
                 score += int(tag.replace("score_", ""))
         self.save_metadata("quiz_score", score)
 
+        self.save_metadata("quiz_sequence", metadata["quiz_sequence"] + 1)
+        return await self.go_to_state("state_quiz_answer_feedback")
+
+    async def state_quiz_answer_feedback(self):
+        metadata = self.user.metadata
+        page_id = metadata["selected_answer_id"]
+        error, page_details = await contentrepo.get_page_details(self.user, page_id, 1)
```

Comment:
Looks like we're doing this call in `state_quiz_process_answer` too, maybe we can put the body and image path in metadata then we don't have to call it again here